packages/amplify-e2e-tests/src/__tests__/container-hosting.test.ts

```diff
@@ -0,0 +1,31 @@
+import { createReactTestProject } from 'amplify-e2e-core';
+
+import { initJSProjectWithProfile, deleteProject } from 'amplify-e2e-core';
+import { enableContainerHosting, addDevContainerHosting, removeHosting } from 'amplify-e2e-core';
+import { deleteProjectDir, getBackendAmplifyMeta } from 'amplify-e2e-core';
```

Comment:
nit: please combine these imports. In VSCode, right-click the code => source action => organize imports